HHS/Head-Start-TTADP
# Production deploy delete non-approved reports

## Description of change

* Non-approved Activity Reports can now be deleted from the alerts section (https://ocio-jira.acf.hhs.gov/browse/TTAHUB-4)
* Users with only `APPROVE_REPORTS` permissions can view all Approved reports in that region. (https://ocio-jira.acf.hhs.gov/browse/TTAHUB-59)


## How to test

1. Sign in as a user with only `APPROVE_REPORTS` permissions. View an approved report from the table. Note that you can successfully see the details
2. As a user with `READ_WRITE_REPORTS` permissions, create a new draft report. Select the context menu from the alerts table and select "Delete report"


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
